User-Help/Platform-Help/Writing-Posts/CodidactAccessibilityHelpArticle.md

```diff
@@ -0,0 +1,255 @@
+
+<h3 id="Animations">Animations</h3>
+
+We don't currently support any way to disable or pause animations in posts, so avoid using animations where possible. In particular, make sure to avoid flashing content (especially anything flashing more than three times a second - don't do that!). Flashing content can cause seizures, and looping animations can be distracting for everyone, but especially for people with some cognitive disabilities.
+
+<h3 id="Text-images">Images of text</h3>
+
+* Avoid images of text.  
+
+  Images of text can't have the text selected, be read by screen readers, indexed by search engines, have the text adjust in a responsive design, or have the font changed. This includes images of code; instead, put the actual code in your post and format it using the dedicated code formatting Markdown.
+
+* If you are quoting from somewhere, use blockquote formatting (which can be done by putting a `>` at the beginning of a paragraph) and cite your source.  
+
+  This applies to both online and offline resources, such as Wikipedia or a physical book.  
+
+* Do not use code formatting for quotes. 
+
+<h2 id="Links">Links</h2>
+
+<h3 id="Link-text">Link text</h3>
+
+* Avoid link text such as "Here" or "Read more".  
+
+  The link should explain its purpose through the text itself. Remember that screen readers and similar tools can jump to specific links, but if they're named something like "this", navigating to the correct link is much harder.  
+Don't go too far in the other direction, though; there's no need to make an entire sentence a link as long as the link text is descriptive and distinct.  
+
+* In general, link text should be unique - don't use the same link text twice in one post if those links go to different places.
+
+<h3 id="Web-Archive">Stable resource (Web Archive)</h3>
+
+While not required by any standard that I'm aware of, I'd encourage you to take steps to make sure that any resource you link to remains stable by archiving it in the Web Archive when you link to it. (This is similar to what Wikipedia does; sources used in articles are almost always archived so that a backup exists.)
+
+<h2 id="Understandable">Making your posts understandable</h2>
+
+<h3 id="Vocabulary">Vocabulary</h3>
+
+* In general, try to keep your vocabulary simple.
+
+  This doesn't mean avoiding all technical terms, or not using the correct terms for things, but don't use jargon or fancy words when it's not necessary. This makes it easier for people who don't speak English as their first language, or people with cognitive disabilities, to understand your post.
+
+This should not come at the expense of precision or accuracy, though; keep your audience in mind. If you are writing for a highly specialized or technical audience, you shouldn't necessary shy aware from using the relevant terminology; but if you're writing for everyone, using relatively simple terms is often a good idea.
```

Comment:
Typo: "shy aware" -> "shy away"

Was this meant to be part of the bullet text (indented)?  (I'm not expressing an opinion; it could go either way.)

Reply:
Typo fixed. The indentation was intentional.